### PR TITLE
Adjustments to allow for migration from amido to ensono azure devops

### PR DESCRIPTION
**Pull request by zecarrera**
#### 📲 What

Removes references to amido
Adjusts variables and YAML files to run on Ensono's ado
fixes linting errors

#### 🤔 Why
		
Migration from amido to ensono ADO.

#### 👀 Evidence
		
The old pipeline running on amido has been [disabled](https://amido-dev.visualstudio.com/Amido-Stacks/_build?definitionId=203)
- New pipeline is running [successfully](https://dev.azure.com/ensonodigitaluk/Stacks/_build?definitionId=25) 
		 

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: build/azDevOps/azure/agent-config-vars.yml [hunk 1]
```diff
@@ -1,10 +1,4 @@
 variables:
-
   # Agent configuration
   - name: pool_vm_image
     value: ubuntu-22.04
-  - name: TaskctlVersion
-    value: 1.4.2
-  - name: StacksEnvfileVersion
-    value: 0.0.16
```
Comment (ElvenSpellmaker) [inline build/azDevOps/azure/agent-config-vars.yml:9]: Why are these being removed?

Comment (zecarrera) [reply build/azDevOps/azure/agent-config-vars.yml:9]: I believe it was because taskCtlVersion was already being defined on setup.yml

Comment (ElvenSpellmaker) [reply build/azDevOps/azure/agent-config-vars.yml:9]: @zecarrera I've pushed a change to standardise the passing of the version and put it back into the `agent-config-vars` file

**Review by ElvenSpellmaker — APPROVED** [review]

**Pull request merged**